src/ui/screens/RegistrationForm.tsx

```diff
@@ -67,54 +78,23 @@ export default class RegistrationForm extends React.Component<Props, State> {
               container
               direction="row"
               justify="space-between">
-              <Grid item
-                    xs={6}
-                    md={5}>
-                <Input
-                  onChange={inputField => this.handleInputChange(inputField)}
-                  fullWidth
-                  value={this.props.credentialCreationData.firstName}
-                  name="firstName"
-                  id="firstName"
-                  placeholder="First Name"
-                />
-              </Grid>
-              <Grid item
-                    xs={6}
-                    md={5}>
-                <Input
-                  onChange={inputField => this.handleInputChange(inputField)}
-                  fullWidth
-                  value={this.props.credentialCreationData.lastName}
-                  name="lastName"
-                  id="lastName"
-                  placeholder="Last Name"
-                />
-              </Grid>
-              <Grid item
-                    xs={6}
-                    md={5}>
-                <Input
-                  onChange={inputField => this.handleInputChange(inputField)}
-                  value={this.props.credentialCreationData.birthDate}
-                  fullWidth
-                  name="birthDate"
-                  id="birthDate"
-                  placeholder="Date of Birth"
-                />
-              </Grid>
-              <Grid item
-                    xs={6}
-                    md={5}>
-                <Input
-                  onChange={inputField => this.handleInputChange(inputField)}
-                  value={this.props.credentialCreationData.nationalId}
-                  fullWidth
-                  name="nationalId"
-                  id="nationalId"
-                  placeholder="National ID"
-                />
-              </Grid>
+              {_.keys(this.props.credentialCreationData).map(
+                (field: any, idx: any) => {
+                  if (field != "faceImage") {
```

Comment:
This isn't a blocker for now but something to keep in mind for the future. We don't want to rely on the field name as much as a configuration that indicates the type of the field.

Reply:
Okay, I addressed this issue.